package.json

```diff
@@ -62,6 +63,7 @@
   "devDependencies": {
     "@babel/core": "^7.14.3",
     "@babel/plugin-transform-react-jsx": "^7.14.3",
+    "@bpmn-io/element-template-chooser": "^0.0.5",
```

Comment:
I cherry-picked this commit inside https://github.com/bpmn-io/bpmn-js-properties-panel/pull/641, so there might be merge conflicts now. Sorry for the inconvenience!

Reply:
No Problem, I have to rebase this anyway to squash my commits :)